postgres_ffi/src/non_portable.rs

```diff
@@ -0,0 +1,234 @@
+//!
+//! This file contains data structures that are explicitly non-portable.
+//! They are needed for compatibility with upstream postgres, which does
+//! not guarantee portability of data files across CPU architectures.
+//!
+//! These data structures use native endian and alignment in postgres.
+//!
+//! This code is not currently guaranteed to work on architectures other
+//! than x86_64.
+//!
+//! Do not use this file, or copy any of the patterns here, for things
+//! that are meant to be portable (particularly data structures sent)
+
```

Comment:
That applies to pretty much all Postgres structs. I think we can move this notice to `postgres_ffi/README`.

Reply:
And rename this file to `pg_control.rs`. All the stuff that's here currently is related to the control file, similar to `vendor/postgres/src/include/catalog/pg_control.h`

Reply:
> That applies to pretty much all Postgres structs. I think we can move this notice to postgres_ffi/README.

It would not apply to postgres network messages, right?

Reply:
That's right, but they are not in the `postgres_ffi` module. The network message are also stable across different PostgreSQL versions.